jbmc/src/java_bytecode/README.md

```diff
@@ -41,9 +41,32 @@ To be documented.
 
 To be documented.
 
-\subsection java-bytecode-genereic-substitution Generic substitution
-
-To be documented.
+\subsection java-bytecode-generic-substitution Generic substitution
```

Comment:
I'd use the term used in the language specification, i.e. 'specialization'.

Reply:
Done